### DISCUSSION TITLE
Is "... the universe regulated by mathematical laws" ?

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#118] [Metaphysics and Philosophy]
"**Newton, Sir Isaac:** (1642-1727) showed by his theory of gravitation that the universe was regulated by mathematical laws." $^{[1]}$

Wrong!  It's the way around: Newton made up the mathematics to describe gravitation.

---
[1] From the "*Glossary of proper names*" in the book "*The ascentric labirynth: Giordano Bruno's prelude to contemporary cosmology*"

**Comment by khuramonline** [comment]
[Is Universe written in Mathematics?](https://bbtrejected.wordpress.com/2019/06/28/is-universe-written-in-mathematics/)

I have this brief blog entry. I quote it below:

We live in a physical world that behaves mathematically. We do not live in a mathematical world that manifests itself physically.

Physical reality comes first. [Equations do not govern physical reality](http://bbtrejected.wordpress.com/2019/06/09/scientists-are-under-delusion-and-they-do-think-that-universe-is-governed-by-their-theories/). The physical reality can be approximated into the form of equations. Physical reality also can be approximated into the form of logical propositions like Newton’s first and third laws of motion. [Newton has described even second law also in the form of logical propositions and without mathematical formulation](http://jamesbrennan.org/physics/notes/Force/axioms.htm).

**Reply by HanDeBruijn** [reply]
>Newton has described even second law also in the form of logical propositions and without mathematical formulation

Wrong. Newton has described his laws with the mathematics of Euclidean geometry; the Principia is full of geometry drawings. Newton has __not__ described his laws with the mathematics of Calculus. Of course not: he was the first one to discover / invent it.

**Reply by khuramonline** [reply]
This is second law from translation of Principia. As far as all three laws are concerned, there is no mathematical formulation and also there is no geometry...!!!

![image](https://github.com/a-cosmology-group/acg/assets/132253587/41491a6b-0d36-420a-966c-3522c247efcd)

**Reply by budrap00** [reply]
Physical reality has order and structure. We are physical components of physical reality. Our concepts of qualitative and quantitative logic derive from our observations of physical reality. Our observations of physical reality are incomplete and always will be by the order and structure of physical reality. Therefore our logical concepts (inclusive of math) are incomplete and always will be.

**Reply by HanDeBruijn** [reply]
@khuramonline . Here are the pictures accompanying your celebrated Theorems XXXI and XXXIII.
![XXXI](https://github.com/a-cosmology-group/acg/assets/135433222/ecae9536-4ed3-4f10-84c1-139740962acf)
![XXXIII](https://github.com/a-cosmology-group/acg/assets/135433222/43596f32-9176-4a9c-b377-dee32db68b8c)
Such as abundant in Newton's Principia. Isn't that classical Euclidean Geometry? Ipse est Mathematics!

**Reply by khuramonline** [reply]
Following are my words in previous message:

> As far as **all three laws are concerned**....

And I know there are lots of geometric diagrams and geometric propositions in Principia. Previously I talked only about the three laws. Now I talk about whole of Principia. In whole of Principia, there are only few geometric propositions or other expressions that can be qualified as "Equations".

**Comment by HanDeBruijn** [comment]
### [The 100 Year Wrong Turn in Cosmology](http://www.conspiracyoflight.com/Cosmology/BigBang1.html)

### The 100 Year Wrong Turn in Physics:
[Quantum Electrodynamics built on a Lie](https://www.youtube.com/watch?v=xaC_aKqjCXU)
[General Relativity, blessing or curse?](https://github.com/orgs/a-cosmology-group/discussions/78)
[The Standard Model](https://home.cern/science/physics/standard-model)

### The 100 Year Wrong Turn in Mathematics:
[English chapters from](http://www.alternatievewiskunde.nl/engels.htm) Waarom is de wiskunde zo onwerkelijk
[_The_ set of _all_ Naturals does not exist](http://www.alternatievewiskunde.nl/MijnBoek/set_theory/ZFbewijs.htm)
[Transfinity](https://www.hs-augsburg.de/~mueckenh/Transfinity/Transfinity/pdf), [Set theory](https://en.wikipedia.org/wiki/Set_theory)

### The 100 Year Wrong Turn in ... Society?
